documentation/README.md

```diff
@@ -4,14 +4,19 @@ These plugins can be used by performance analysis applications that use the SDK
 
 This folder contains instructions for developing SDK plugins and utilizing the SDK's various features.
 
+# Quick Start
+Refer to [Creating your first plugin](Using-the-SDK/Creating-your-plugin.md) to jump in quickly.
+
+> ⚠️ We recommend following the [Recommended Reading Order](#recommended-reading-order).
```

Comment:
Wondering if it might just be better to put the recommended reading order in here and deleting that Header below.

Reply:
I like it better as two different sections as they have different reasons to be.